boards/arm/adafruit_grand_central_m4_express/doc/index.rst

```diff
@@ -0,0 +1,191 @@
+.. _adafruit_grand_central_m4_express:
+
+Adafruit Grand Central M4 Express
+#################################
+
+Overview
+********
+
+The Adafruit Grand Central M4 Express is an ARM development board with the
+form factor of an Arduino Mega.
+It features 70 GPIO pins, a microSDHC slot and 8MiB of QSPI Flash.
+
+.. image:: img/adafruit_grand_central_m4_express.jpg
+     :align: center
+     :alt: Adafruit Grand Central M4 Express
```

Comment:
```suggestion
     :alt: Adafruit Grand Central M4 Express

     Adafruit Grand Central M4 Express (Credit: XXXX)
```

Reply:
IANAL but this is basically fair use of the original image, and adding the credit mention is an additional way to make it clear that we are indeed just using the image for illustration purposes, nothing else.